packages/devtools_app/lib/src/file_import.dart

```diff
@@ -99,13 +99,14 @@ class _FileImportContainerState extends State<FileImportContainer> {
   }
 
   Widget _buildImportFileRow() {
-    const rowHeight = 37.0;
+    // TODO(jacobr): why is this 37.0 instead of a more standard height?
+    final rowHeight = scaleByFontFactor(37.0);
```

Comment:
@kenzieschmoll any idea why we used this odd height?

Reply:
I do not know. Feel free to change to a more standard height as long as we verify the layout still looks good